vm/kvm-unit-tests/runtest.sh

```diff
@@ -48,198 +54,285 @@ function check_virt_support
     fi
 }
 
-# test is only supported on x86_64, aarch64, ppc64 and s390x
-hwpf=$(uname -i)
-check_platform_support $hwpf
-if (( $? == 0 )); then
-    echo "Running on supported arch ($hwpf)" | tee -a $OUTPUTFILE
+function getTests
+{
+    # List of tests to run on all architectures
+    ALL_TESTS=()
+    while IFS=  read -r -d $'\0'; do
+        ALL_TESTS+=("$REPLY")
+    done < <(find ${BINDIR}/ -maxdepth 1 -type f -executable -printf "%f\0")
+}
+
+function disableTests
+{
+    typeset hwpf=$(uname -i)
+
+    # Disable tests for RHEL8 Kernel (4.18.X)
+    if uname -r | grep --quiet '^4'; then
+        # Disabled x86_64 tests for Intel & AMD machines due to bugs
+        if [[ $hwpf == "x86_64" ]]; then
+            # Disable test hyperv_synic
+            # due to https://bugzilla.redhat.com/show_bug.cgi?id=
```

Comment:
More empty bugzilla ids. I'm starting to think it's on purpose, but I don't understand it 😓